packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.gql.js

```diff
@@ -0,0 +1,48 @@
+import { gql } from '@apollo/client';
+import { AccountInformationPageFragment } from './accountInformationPageFragment.gql';
+
+export const SET_CUSTOMER_INFORMATION = gql`
+    mutation SetCustomerInformation($customerInput: CustomerInput!) {
+        updateCustomer(input: $customerInput)
+            @connection(key: "updateCustomer") {
+            customer {
+                ...AccountInformationPageFragment
```

Comment:
Need to add `id` field here. It seems redundant but the linter can't assume the provided fragment has `id` in it.

Reply:
I added that.